pinn/visualizer.py

```diff
@@ -24,14 +24,25 @@ def VgVd_counter (vg, vd):
 
     return vg_count, vd_count
 
-def plot_data (tag, x, y, tag_count, x_count, line_style):
+def plot_data (tag, x, y, tag_count, x_count, line_style, y_scale):
     i = 0
-    while (i < tag_count):
-        x_tmp = x[i*x_count : (i+1)*x_count]
-        y_tmp = np.abs(y[i*x_count : (i+1)*x_count])
-        # vg_label = 'vg='+str(vg[i*vd_count])
-        plt.plot(x_tmp, y_tmp, ls = line_style, color = colors.cnames.keys()[i])
-        i = i+1
+    if (y_scale == 'log'):
+        while (i < tag_count):
+            x_tmp = x[i*x_count : (i+1)*x_count]
+            y_tmp = np.abs(y[i*x_count : (i+1)*x_count])
+            # vg_label = 'vg='+str(vg[i*vd_count])
```

Comment:
Can you eliminate some duplicate code by putting if statement around y_tmp only?